rootfs/standard/usr/share/mynode_apps/albyhub/albyhub.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "albyhub",
+    "short_name": "albyhub",
+    "author": {
+        "name": "getAlby",
+        "link": "https://www.albyhub.com"
+    },
+    "website": {
+        "name": "GitHub",
+        "link": "https://github.com/getAlby/hub"
+    },
+    "category": "Bitcoin",
+    "short_description": "Alby Hub is a self-custodial, open source lightning wallet that connects to apps",
+    "description": [
+        "Lightning Made Easy - Running your own node has never been easier.",
+        "Set up your own node in one click, top up channels via credit card or bank transfer",
+        "choose from the most reliable liquidity providers in the space",
+        "and get 24/7 live support – all in one intuitive interface.",
+        "",
+        "You will also easily understand how the lightning network works",
+        "learning effortlessly along the way."
+    ],
+    "latest_version": "v1.15.0",
+    "supported_archs": null,
+    "download_skip": false,
+    "download_type": "source",
```

Comment:
What should be download type? Cause "source" now loads tar.gz which is not used at all as we run docker?

Reply:
You can set "download_skip" to true.